sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -44,7 +44,7 @@ import org.apache.spark.sql.catalyst.trees.{AlwaysProcess, CurrentOrigin}
 import org.apache.spark.sql.catalyst.trees.CurrentOrigin.withOrigin
 import org.apache.spark.sql.catalyst.trees.TreePattern._
 import org.apache.spark.sql.catalyst.util.{toPrettySQL, CharVarcharUtils}
-import org.apache.spark.sql.connector.catalog._
+import org.apache.spark.sql.connector.catalog.{View => _, _}
```

Comment:
Is this change needed? It looks suspicious

Reply:
Compile error without the change:
```
[ERROR] /Users/jzhuge/Repos/upstream-spark/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala:1002: reference to View is ambiguous;
it is imported twice in the same scope by
import org.apache.spark.sql.connector.catalog._
and import org.apache.spark.sql.catalyst.plans.logical._
```

Reply:
Got it. This avoids a conflicting import. Thanks!